lusSTR/scripts/marker.py

```diff
@@ -1511,7 +1511,9 @@ def designation(self):
             if match:
                 length = int(match.group(1))
                 match_list.append(length)
-        if len(match_list) == 1:
+        if len(match_list) == 0:
+            lus, sec, ter = 0, 0, 0
+        elif len(match_list) == 1:
```

Comment:
While running some test data, I ran into an error when lusSTR encountered a short sequence from the DYS448 locus. While this likely won't be encountered often, it prevents lusSTR from breaking.